tests/lib/wheel.py

```diff
@@ -243,7 +249,7 @@ def record_file_maker_wrapper(
     with StringIO(newline="") as buf:
         writer = csv23.writer(buf)
```

Comment:
Note for follow-up: we should drop `csv23` from tests and use the built-in `csv` instead.